src/extension/background-script/state.ts

```diff
@@ -133,6 +137,21 @@ const state = createState<State>((set, get) => ({
     const connector = await connectorPromise;
     return connector;
   },
+  getLiquid: async () => {
+    const currentLiquid = get().liquid;
+    if (currentLiquid) {
+      return currentLiquid;
+    }
+    const mnemonic = await get().getMnemonic();
+    const currentAccountId = get().currentAccountId as string;
+    const account = get().accounts[currentAccountId];
+
+    const isMainnet = account.bitcoinNetwork === "bitcoin";
+
+    const liquid = new Liquid(mnemonic, isMainnet ? "liquid" : "testnet");
```

Comment:
we have this check in a few places, it would be nice to move to one utility function if possible (in case we support regtest later)

Reply:
Fixed in https://github.com/getAlby/lightning-browser-extension/pull/2583